src/workers/cdc/primary.ts

```diff
@@ -663,6 +665,17 @@ const onSourceChange = async (
     if (data.payload.before!.private !== data.payload.after!.private) {
       await notifySourcePrivacyUpdated(logger, data.payload.after!);
     }
+
+    const beforeFlags = data.payload.before.flags as unknown as string;
+    const afterFlags = data.payload.after.flags as unknown as string;
+    const before = JSON.parse(beforeFlags || '{}') as SourceFlagsPublic;
+    const after = JSON.parse(afterFlags || '{}') as SourceFlagsPublic;
+    if (before.featured !== after.featured && after.featured) {
```

Comment:
Should we be more explicit and say `after.featured === true`?
As the worked only expects truthy calls?